src/it/test14-primary/verify.groovy

```diff
@@ -0,0 +1,5 @@
+def pb = new ProcessBuilder ( "bash", "-c", "rpm -qlvvp target/*.rpm" );
```

Comment:
As you want to ensure a specific file name and location, I think you should do an actual check for that. You could use a simple groovy check, instead of using the `rpm` command here. This would make things easier for testing locally on non-RPM platforms.

You can find an example in the `test10-defaultname` test.

Reply:
oh thats way simpler!  I had just grabbed test1 as my starting point, but test10 is better thanks!